### PR TITLE
Feature: Login modules

### PR DESCRIPTION
**Pull request by JakubFrackowiak**
Sign up:

image

Sign in:

image

Forgot password:

image

Reset password:

image

Verify phone number:

image

### CODE REVIEW

**Comment by JakubFrackowiak** [comment]
I will fix build today, little busy rn

File: components/SignUp/VerifyPhoneNumber.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        height={70}
+        alt="Alert circle"
+      />
+      <Typography variant="h3">Check your phone</Typography>
+      <Stack alignItems="center">
+        <Typography variant="h6" color="text.secondary">
+          We sent a verification code to
+        </Typography>
+        <Typography variant="h6" color="text.secondary" fontWeight="600">
+          +48 123 456 789
```
Comment (piotrostr) [inline components/SignUp/VerifyPhoneNumber.tsx:20]: this should be the actual number

File: components/SignUp/EmailSignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+            console.log(response)
+          },
+        },
+        auth
+      )
+    )
+    const code = window.prompt("Enter OTP", "")
+    const result = await confirmationResult.confirm(code)
+    console.log(result)
+    // TODO this needs try/catch/finally blocks as in SignIn.tsx
```
Comment (piotrostr) [inline components/SignUp/EmailSignUp.tsx:111]: this is unresolved

File: components/SignIn/EmailSignIn.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+export function EmailSignIn() {
+  const [severity, setSeverity] = useState<AlertColor>("info")
+  const [message, setMessage] = useState("")
+  const [open, setOpen] = useState(false)
+  const auth = useAuth()
+
+  // TODO(JakubFrackowiak) This is not even resetting password
+  async function resetPassword() {
```
Comment (piotrostr) [inline components/SignIn/EmailSignIn.tsx:35]: placeholder method, TODO implement

**Review by piotrostr — CHANGES_REQUESTED** [review]
@JakubFrackowiak

**Review by OliwierOST — APPROVED** [review]
All good for me, let's hear what @piotrostr has to say.

**Review by piotrostr — APPROVED** [review]

**Pull request merged**